packages/lib/src/vis/line/DataCurve.tsx

```diff
@@ -79,8 +105,10 @@ function DataCurve(props: Props) {
         visible={showLine}
         geometry={dataGeometry}
         onClick={onLineClick && handleClick}
+        onPointerOver={onLinePointerEnter && handlePointerEnter}
+        onPointerOut={onLinePointerOut && handlePointerOut}
       >
-        <lineBasicMaterial color={color} linewidth={LINE_WIDTH} />
+        <lineBasicMaterial color={color} />
```

Comment:
I realized that there is not much point specifiying `linewidth` as it is inherently set to `1` anyway: https://threejs.org/docs/#api/en/materials/LineBasicMaterial.linewidth